mlir/lib/Dialect/Tosa/IR/TosaOps.cpp

```diff
@@ -947,6 +1063,18 @@ LogicalResult tosa::PadOp::inferReturnTypeComponents(
 }
 
 LogicalResult tosa::PadOp::verify() {
+  if (verifySameElementTypes(*this, /* inType = */ getInput1().getType(),
+                             /* outType = */ getOutput().getType())
+          .failed()) {
+    return failure();
+  }
+  if (auto padConst = getPadConst()) {
```

Comment:
nit: Leave spaces between the ifs?
Sorry this is quite a nitpick. Feel free to ignore :) 
Same for all the similar cases below

Reply:
sure, easy fix. done.